packages/sui-react-web-vitals/src/index.js

```diff
@@ -58,17 +71,31 @@ export default function WebVitalsReporter({
       return deviceType || browser?.deviceType
     }
 
-    const handleReport = ({name, value}) => {
+    const handleAllChanges = ({attribution, name, value}) => {
+      const amount = name === METRICS.CLS ? value * 1000 : value
+      const pathname = getPathname()
+      const isExcluded =
+        !pathname || (Array.isArray(pathnames) && !pathnames.includes(pathname))
+
+      if (isExcluded || !logger?.log || amount < thresholds[name]) return
+
+      logger.log(
+        JSON.stringify({
+          name: `cwv|${name.toLowerCase()}`,
```

Comment:
`cwv.${name.toLowerCase()}` 🙏🏻 let's always use dot notation for naming log/metric events